core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -489,14 +487,13 @@ void writeValue(int hi, V value) {
      * turn has the value. For linkage this is an insert linearization point.
      * All the relevant data can be found inside opData.
      *
-     * point to value
-     * if unsuccessful this means someone else got to it first (helping rebalancer or other operation)
+     * if someone else got to it first (helping rebalancer or other operation), returns the old handle
      */
-    boolean pointToValue(OpData opData) {
+    Handle pointToValue(OpData opData) {
```

Comment:
I'll get deeper into it once the code is merged with the last (and final) Yoni changes